.github/workflows/ci.yml

```diff
@@ -77,26 +77,43 @@ jobs:
         include:
           - machine: windows-2022
             distribution: windows
+            log-dir: "/c/ProgramData/OpenTelemetry .NET AutoInstrumentation/logs"
           - machine: ubuntu-20.04
             distribution: linux-glibc
+            log-dir: "/var/log/opentelemetry/dotnet"
           - machine: ubuntu-20.04
             distribution: linux-musl
+            test-instrument: "false"
```

Comment:
Why is this false here?

Reply:
I think that `linux-musl` should be tested on Alpine. I will try to test the Alpine in next PR by using `build/nuke/docker/Alpine.dockerfile`